includes/cs-style-option.php

```diff
@@ -0,0 +1,1170 @@
+
+$cswp_icon = ( ! empty( $cswp_value['cswp_icon'] ) ? $cswp_value['cswp_icon'] : 'emptyimg.jpg' );
+
+$cswp_button_hide = get_option( 'cswp_form_data' );
+if ( 'button' === $cswp_button_hide['cswp_button_type'] ) {
+	$cswp_button_css   = 'style="display:block"';
+	$cswp_symbol_css   = 'style="display:none"';
+	$cswp_toggle_css   = 'style="display:none"';
+	$cswp_dropdown_css = 'style="display:none"';
+} elseif ( 'toggle' === $cswp_button_hide['cswp_button_type'] ) {
+	$cswp_toggle_css   = 'style="display:block"';
+	$cswp_symbol_css   = 'style="display:none"';
+	$cswp_button_css   = 'style="display:none"';
+	$cswp_dropdown_css = 'style="display:none"';
+} elseif ( 'dropdown' === $cswp_button_hide['cswp_button_type'] ) {
+	$cswp_dropdown_css = 'style="display:block"';
+	$cswp_symbol_css   = 'style="display:none"';
+	$cswp_toggle_css   = 'style="display:none"';
+	$cswp_button_css   = 'style="display:none"';
+} elseif ( 'symbol' === $cswp_button_hide['cswp_button_type'] ) {
+	$cswp_symbol_css   = 'style="display:block"';
+	$cswp_dropdown_css = 'style="display:none"';
+	$cswp_toggle_css   = 'style="display:none"';
+	$cswp_button_css   = 'style="display:none"';
+} else {
+	$cswp_dropdown_css = 'style="display:block"';
+	$cswp_symbol_css   = 'style="display:none"';
+	$cswp_toggle_css   = 'style="display:none"';
+	$cswp_button_css   = 'style="display:none"';
+}
+
+$currencybtn       = CS_Loader::cswp_load_currency_button_data();
+$base_value_select = CS_Loader::cswp_load_all_data();
+
+if ( ! empty( $currencybtn ) ) {
+	foreach ( $currencybtn as $cswp_base_value ) {
+		if ( $cswp_base_value === $base_value_select['basecurency'] ) {
+			continue;
+		}
+		$curbtn[] = $cswp_base_value;
+	}
+	if ( ! empty( $curbtn ) && is_array( $curbtn ) ) {
+		array_push( $curbtn, $base_value_select['basecurency'] );
+		$currencybtn = array_combine( $curbtn, $curbtn );
+	}
+}
+?>
+<!-- Html code for frontend -->
+<form method="post" name="cca_settings_form">
+	<table class="form-table" >
+	<tr>
+			<td class="cswp_table_row" scope="row">
+				<code class="cswp_code"><label for="CSWP"> <?php esc_attr_e( 'Overall Alignment', 'advanced-currency-switcher' ); ?></label></code>
+			</td>
+	</tr>
+	<tr >
+			<th class="cswp_table_row" scope="row">
+				<label for="CSWPOverAlign"><?php esc_attr_e( 'Alignment', 'advanced-currency-switcher' ); ?></label>
+			</th>
+			<td class="cswp_table_td">
+			<select name="cswp_alignment">
+				<?php
+				if ( 'left' === $cswp_alignment ) {
+
+					echo '<option selected value="left">Left</option>';
+				} else {
+
+					echo '<option  value="left">Left</option>';
+				}
+				if ( 'center' === $cswp_alignment ) {
+
+					echo '<option selected value="center">Center</option>';
+				} else {
+
+					echo '<option  value="center">Center</option>';
+				}
+				if ( 'right' === $cswp_alignment ) {
+
+					echo '<option selected value="right">Right</option>';
+				} else {
+
+					echo '<option  value="right">Right</option>';
+				}
```

Comment:
This block of code can be reduced by the use of selected function - https://developer.wordpress.org/reference/functions/selected/  

Make changes in other places also.